swap_meet/vendor.py

```diff
@@ -1,2 +1,57 @@
+from swap_meet.item import Item 
+
 class Vendor:
-    pass
+    def __init__(self, inventory = []):
+        self.inventory = inventory 
```

Comment:
careful passing in a data structure like a list as a default parameter in a class constructor. Every instance that is made will all use the _same_ list instead of each instance having its own unique inventory list. this is known as a "side effect."

so what we can do to stop this side effect is:

```suggestion
    def __init__(self, inventory = None):
        if inventory == None:
            self.inventory = []
        else:
            self.inventory = inventory 
```

now for every instance of Vendor that is created without a pre-made inventory list, the instance will get its own unique empty list.